core/templates/dev/head/player/PlayerServices.js

```diff
@@ -356,54 +340,16 @@ oppia.factory('oppiaPlayerService', [
       };
     },
     getExplorationTitle: function() {
-      return _exploration.title;
+      return exploration.title;
     },
     getInteractionHtml: function(stateName, labelForFocusTarget) {
       return oppiaExplorationHtmlFormatterService.getInteractionHtml(
```

Comment:
Please see my previous comment regarding how much logic should go in the object factory. Perhaps having some sort of ExplorationRenderService next to the factory (in ``domain/exploration``) would help alleviate some of these concerns. It makes more sense to me to collect all of the render-esque code into a single service than putting them in the object factory.